### ISSUE TITLE
Duplicate poolParamsOwners entry

### ISSUE THREAD

**Issue by SebastienGllmt** [input-output-hk/cardano-db-sync#380]
# Commit where this issue was found

`570945db676548d88b6818af94d926ccfafa55cb`

# Bug

The `poolParamsOwners` entry contains the same value twice for many pools (ex: `1b268f4cba3faa7e36d8a0cc4adca2096fb856119412ee7330f692b5` - DIGI)

# SQL query to reproduce the issue

```sql
select 
  (
   select json_agg(encode(hash,'hex'))
    from pool_owner
    where pool_owner.pool_hash_id = pool_hash.id
  ) as "poolParamsOwners"
from pool_update as pool
join pool_hash
  on pool.hash_id = pool_hash.id
where encode(pool_hash.hash_raw,'hex') = '1b268f4cba3faa7e36d8a0cc4adca2096fb856119412ee7330f692b5'
```

This gives the result

```
                                                     poolParamsOwners
--------------------------------------------------------------------------------------------------------------------------
 ["adfc4a5d637fc74e0ecf763772f9d9adf4ec11cd3c9a1c435ae3ff99", "adfc4a5d637fc74e0ecf763772f9d9adf4ec11cd3c9a1c435ae3ff99"]

 ["adfc4a5d637fc74e0ecf763772f9d9adf4ec11cd3c9a1c435ae3ff99", "adfc4a5d637fc74e0ecf763772f9d9adf4ec11cd3c9a1c435ae3ff99"]

(2 rows)
```

# Expected behavior:

On 5.0.2 we got a single result `adfc4a5d637fc74e0ecf763772f9d9adf4ec11cd3c9a1c435ae3ff99`

# Possible cause

I noticed DIGI has two registration certificates, so it's possible that the `poolParamsOwners` gets duplicated for the second registration.

**Comment by erikd** [comment]
Simplifying the query and then only printing the most relevant bits:
```
cexplorer=# select pool_update.meta, pool_hash.view
            from pool_update inner join pool_hash on pool_update.hash_id = pool_hash.id
            where pool_hash.hash_raw = '\x1b268f4cba3faa7e36d8a0cc4adca2096fb856119412ee7330f692b5' ;
 meta |                           view                           
------+----------------------------------------------------------
  578 | pool1rvng7n968748udkc5rxy4h9zp9hms4s3jsfwuues76ft28uc056
  229 | pool1rvng7n968748udkc5rxy4h9zp9hms4s3jsfwuues76ft28uc056
(2 rows)
```
so these two entries are in fact distinct. Not sure what `5.0.2` was doing and I do have an instance of that running in order to test.

The `meta` field is an index into the `pool_meta_data` table.

If you look at the `pool_hash` table, there is only one entry for that hash:
```
cexplorer=# select * from pool_hash
            where pool_hash.hash_raw = '\x1b268f4cba3faa7e36d8a0cc4adca2096fb856119412ee7330f692b5' ;
 id  |                          hash_raw                          |                           view                           
-----+------------------------------------------------------------+----------------------------------------------------------
 211 | \x1b268f4cba3faa7e36d8a0cc4adca2096fb856119412ee7330f692b5 | pool1rvng7n968748udkc5rxy4h9zp9hms4s3jsfwuues76ft28uc056
(1 row)
```
If this is sufficient investigation/explanation please close this ticket.

**Comment by SebastienGllmt** [comment]
Your answer made me realize the mistake we were making. I think probably an issue with 5.0.2 that got fixed with 6.0.0 and the fix is what made us realize our code had a bug in it. Thanks!
